exonum/src/sandbox/sandbox.rs

```diff
@@ -642,6 +670,16 @@ impl Drop for Sandbox {
     }
 }
 
+impl ConnectList {
+    // Creates from state::peers, needed only for testing.
```

Comment:
I prefer to make this comment a doc-comment (and expand it a little bit) - documentation still can be generated using `strip-hidden.